src/Maestro/Maestro.Web/.config/settings.Development.json

```diff
@@ -1,8 +1,13 @@
 {
+  "HealthTableUri": "",
   "KeyVaultUri": "https://maestrolocal.vault.azure.net/",
   "AppConfigurationUri": "https://maestrolocal.azconfig.io/",
   "BuildAssetRegistry": {
     "ConnectionString": "Data Source=localhost\\SQLEXPRESS;Initial Catalog=BuildAssetRegistry;Integrated Security=true"
   },
-  "ForceLocalApi": false
+  "ForceLocalApi": false,
+  "DataProtection": {
```

Comment:
Why do we have to add all these extra empty entries?

Reply:
The config code that runs in the service just returns empty string for secrets that don't exist, and for the ones I added the service itself is okay with that, or just doesn't use the value at all. I needed these to make the config validation bit not complain that you are using a missing secret.

Reply:
Ah. So we were _relying_ on secrets not existing returning the empty string.  This does seem much safer. :-)